### ISSUE TITLE
CC001 mishandles Condition with Or using mixed case

### ISSUE THREAD

**Issue by DinoChiesa** [apigee/apigeelint#529]
Parsing a condition like 
```
error.status.code = 429 Or error.status.code GreaterThan 399
```

Results in a different AST than parsing of a condition like 
```
error.status.code = 429 or error.status.code GreaterThan 399
```

...because of inappropriate case sensitivity in TruthTable.js .

This leads to spurious warnings about conditions in CC001.
